site/docs/core/bundle.md

```
@@ -0,0 +1,112 @@
+    forceCopyDirs:
+      - app/public
+      - app/assets
+  pack:
+    resolve:
+      alias:
+        some-package: ./node_modules/some-package/index.js
+```
+
+## Output
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)